pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -235,6 +315,32 @@ type PravegaSpec struct {
 	// InfluxDB Secret specifies the secret name containing credentials and mount path volume
 	// that has to be configured in controller and segmentstore pods
 	InfluxDBSecret *InfluxDBSecret `json:"influxDBSecret,omitempty"`
+
+	// Probes specifies the timeout values for the Readiness and Liveness Probes
```

Comment:
Instead of specifying `timeout`, we can make it more general like:
`Controller Probes specifies the values for configurable fields of Readiness and Liveness Probes for the controller pods.   `

Reply:
fixed